python/wipe_appwrite_bucket/main.py

```diff
@@ -0,0 +1,44 @@
+import json
+from appwrite.client import Client
+from appwrite.services.storage import Storage
+
+
+def main(req, res):
+    client = Client()
+
+    print("Starting function")
+
+    # Make sure we have envirnment variables required to execute
+    if not req.variables.get(
+        "APPWRITE_FUNCTION_ENDPOINT", None
+    ) or not req.variables.get("APPWRITE_FUNCTION_API_KEY", None):
+        return res.json(
+            {"success": False, "message": "Required environment variables not found"}
+        )
+
+    payload = json.loads(req.payload)
+
+    if "bucketId" not in payload:
+        return res.json({"success": False, "message": "Missing bucketId"})
+
+    bucket_id = payload["bucketId"]
```

Comment:
It's a good idea to be more defensive here too. If `req.payload` is malformed or doesn't contain `bucketId` this function will throw an exception.

Reply:
I don't get it. I am doing the check if `bucketId` is there in payload or not